container.go

```diff
@@ -77,7 +77,7 @@ type ImageBuildInfo interface {
 	GetDockerfile() string                          // the relative path to the Dockerfile, including the file itself
 	GetRepo() string                                // get repo label for image
 	GetTag() string                                 // get tag label for image
-	ShouldPrintBuildLog() bool                      // allow build log to be printed to stdout
+	BuildLogWriter() io.Writer                      // for output of build log, use io.Discard to disable the output
```

Comment:
bug: as this is breaking change it will require ! and a BREAKING_CHANGE note, see: https://www.conventionalcommits.org/en/v1.0.0/

Reply:
Fixed the name of the PR